src/js/config.js

```diff
@@ -1,24 +1,28 @@
 // Note that we import these values into "web_app_config" (so we can search for it)
```

Comment:
Hi @SailingSteve, can you please remove src/js/config.js from your pull request? We unfortunately need to leave the config.js file in the repo so that automated tests run, but we don't want to check in our changes.